strawberry/types/fields/resolver.py

```diff
@@ -190,6 +187,18 @@ def __call__(self, *args: str, **kwargs: Any) -> T:
     def signature(self) -> inspect.Signature:
         return Signature.from_callable(self._unbound_wrapped_func, follow_wrapped=True)
 
+    # TODO: find better name
```

Comment:
Not sure about this name, seems a bit silly to have `annotations` and `strawberry_annotations` even though `strawberry_annotations` includes reserved parameters as well unlike `annotations`. Could we consider removing the `annotations` property below @patrick91 ? Any ideas welcome!

Reply:
Could we just use annotations?

Reply:
Do you mean just get rid of this and use the annotations property below?

Reply:
yeah, I was suggesting that 😊 do you think it is worth doing it?

Reply:
@patrick91 I don't think it's worth it to go back to how it was implemented
previously with the resolved annotations being injected by `ReservedType`, it's a nicer flow to create the `StrawberryAnnotation` instances
first. Maybe we can consider replacing the current `annotation` property below?
It would be a breaking change though.

Reply:
let's do that in future, if we remember ;)